esiclient/v1/mdc/mdc_node_baremetal.py

```diff
@@ -21,6 +21,7 @@ class MDCBaremetalNodeList(command.Lister):
     """List baremetal nodes from multiple OpenStack instances"""
 
     log = logging.getLogger(__name__ + ".List")
+    auth_required = False
```

Comment:
How does setting the variable here even do anything?

Reply:
It's the same as setting the log variable above it; it just sets a parameter which the parent class uses.